web/lumpTypeView.ts

```diff
@@ -471,3 +471,62 @@ function drawMapGeometry(
         }
     }
 }
+
+export function LumpTypeViewPlaypal(
+    options:{scaleX:number, scaleY:number} = {scaleX: 4, scaleY: 4}
```

Comment:
I think that this could be better and more readable. I don't like having `:` without a space between the identifier and the type because stuff gets crammed together. I also don't know that it is necessary to have an options object here if the argument is never used, and particularly if there are only two options. Perhaps change this function signature to this instead?

``` ts
export function LumpTypeViewPlaypal(scaleX: number = 4, scaleY: number = 4): LumpTypeView {
```